packages/theming/theming-utils/src/mapPipelineToTheme.ios.ts

```diff
@@ -1,134 +1,116 @@
-    neutralBackground5Hover: pipelineOutput.neutralBackground5.fillColorHover,
     neutralBackground5Pressed: pipelineOutput.neutralBackground5.fillColorPressed,
     neutralBackground5Selected: pipelineOutput.neutralBackground5.fillColorSelected,
+
     neutralBackground6: pipelineOutput.neutralBackground6.fillColorRest,
-    neutralBackgroundInverted: pipelineOutput.neutralBackgroundInverted.fillColorRest,
 
-    subtleBackground: pipelineOutput.subtleBackground.fillColorRest,
-    subtleBackgroundHover: pipelineOutput.subtleBackground.fillColorHover,
-    subtleBackgroundPressed: pipelineOutput.subtleBackground.fillColorPressed,
-    subtleBackgroundSelected: pipelineOutput.subtleBackground.fillColorSelected,
+    neutralBackgroundCanvas: pipelineOutput.neutralBackgroundCanvas.fillColorRest,
+
+    neutralBackgroundDarkStatic: pipelineOutput.neutralBackgroundDarkStatic.fillColorRest,
+
+    neutralBackgroundLightStatic: pipelineOutput.neutralBackgroundLightStatic.fillColorRest,
 
-    transparentBackground: pipelineOutput.transparentBackground.fillColorRest,
-    transparentBackgroundHover: pipelineOutput.transparentBackground.fillColorHover,
-    transparentBackgroundPressed: pipelineOutput.transparentBackground.fillColorPressed,
-    transparentBackgroundSelected: pipelineOutput.transparentBackground.fillColorSelected,
+    neutralBackgroundLightStaticDisabled: pipelineOutput.neutralBackgroundLightStaticDisabled.fillColorRest,
+
+    neutralBackgroundInverted: pipelineOutput.neutralBackgroundInverted.fillColorRest,
 
     neutralBackgroundDisabled: pipelineOutput.neutralBackgroundDisabled.fillColorRest,
 
     neutralStencil1: pipelineOutput.neutralStencil1.fillColorRest,
+
     neutralStencil2: pipelineOutput.neutralStencil2.fillColorRest,
 
-    brandBackground: pipelineOutput.brandBackground.fillColorRest,
-    brandBackgroundHover: pipelineOutput.brandBackground.fillColorHover,
-    brandBackgroundPressed: pipelineOutput.brandBackground.fillColorPressed,
-    brandBackgroundDisabled: pipelineOutput.brandBackground.fillColorDisabled,
-    brandBackgroundSelected: pipelineOutput.brandBackground.fillColorSelected,
-    compoundBrandBackground1: pipelineOutput.compoundBrandBackground1.fillColorRest,
-    compoundBrandBackground1Hover: pipelineOutput.compoundBrandBackground1.fillColorHover,
-    compoundBrandBackground1Pressed: pipelineOutput.compoundBrandBackground1.fillColorPressed,
+    /// Neutral foreground tokens
 
-    brandBackgroundStatic: pipelineOutput.brandBackgroundStatic.fillColorRest,
-    brandBackground2: pipelineOutput.brandBackground2.fillColorRest,
+    neutralForeground1: pipelineOutput.neutralForeground1.fillColorRest,
+
+    neutralForeground2: pipelineOutput.neutralForeground2.fillColorRest,
+
+    neutralForeground3: pipelineOutput.neutralForeground3.fillColorRest,
+
+    neutralForegroundDisabled1: pipelineOutput.neutralForegroundDisabled1.fillColorRest,
```

Comment:
Re-looking a the tokens, I think `neutralForegroundDisabled1` can be completely removed to use `neutralForegroundDisabled` for all endpoints ( or reverse). They are present in both endpoints with different names, and could be merged already.

```
/** @platform macOS, win32, windows */
  neutralForegroundDisabled?: ColorValue;

  /** @platform android, iOS */
  neutralForegroundDisabled1?: ColorValue;

```

Reply:
Thanks for pointing this out - asked about how to resolve this inconsistency on Teams. I think the designs themselves and the design-tokens repo should be updated to be consistent across platforms to avoid confusion

Reply:
Going to leave as is for now and follow up once this has been discussed on Teams